packages/ui-react/src/components/Shelf/Shelf.tsx

```diff
@@ -10,8 +11,8 @@ import ChevronLeft from '@jwp/ott-theme/assets/icons/chevron_left.svg?react';
 import ChevronRight from '@jwp/ott-theme/assets/icons/chevron_right.svg?react';
 import useBreakpoint, { Breakpoint, type Breakpoints } from '@jwp/ott-ui-react/src/hooks/useBreakpoint';
 import type { PosterAspectRatio } from '@jwp/ott-common/src/utils/collection';
+import '@videodock/tile-slider/lib/style.css';
```

Comment:
Hm, why can't we use styles from `TileSlider` itself?

Reply:
This is using the styles from TileSlider, right? 😅

Reply:
Right, I mean, can we import them in `TileSlider.tsx` and get them from component and not separately?
Or is your idea to redefine them for some use cases?